### ISSUE TITLE
Content inside Visual Studio Code popup does not adapt after applying resize settings: A11y_VisualStudioCode_Resize

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#307579]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier." 

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".

## GitHub Tags: 
#A11yTCS;#Visual Studio Code Client; #E2E_VisualStudioCodeClient_Win32_Dec2025;#DesktopApp;#FTP;#Win32;#A11ySev2;#A11yMAS;#WCAG1.4.4; #ResizeText; #Closed; 

### Prerequisites:

1. Set the display resolution to 1366*768
2. Set the text size at 200

## Environmental Details: 
App Name: Visual Studio Code Client 
Microsoft Windows 11 Enterprise 
OS Build:  26200.7392

## Repro Steps: 

1. Open Visual Studio Code Insiders editor
2. Install the Microsoft Sentinel extension
3. TAB till Microsoft Sentinel in left navigation and press ENTER key.
4. TAB till Sign in to view lake tables and press ENTER key.
5. Visual studio code popup will open observe the issue.

## Actual Results: 
After applying resize settings, the content inside a Visual Studio Code popup does not adapt to the resized viewport.

## Expected Results: 
When resize settings are applied, All popup content should resize correctly within the popup container.

## User Impact: 
Low‑vision users who rely on text resizing  cannot fully read or access popup content.

### Attachment:

<img width="1280" height="768" alt="Image" src="https://github.com/user-attachments/assets/c40b46c0-101b-481b-a620-5f8fd20ce965" />

**Comment by meganrogge** [comment]
This dialog is provided by the native Windows API, so its sizing and zoom behavior are controlled by the OS, not VS Code.

Users can zoom it using system-level tools like Windows Magnifier (Win + +) or by adjusting display scaling, but there isn’t an app-level way for us to modify its zoom behavior.

If there are issues with scaling or blurriness, that could point to DPI awareness mismatches, but otherwise this is expected behavior for native dialogs.

I suggest closing this.

**Comment by kapilvaishna** [comment]
@meganrogge Please don't close this issue, I will follow with external team to fix this issue as per external bug process. 
Could you please le us know under which service of Window, this issue will come.

**Comment by meganrogge** [comment]
I do not know which service, sorry.

**Comment by kapilvaishna** [comment]
Issue is not repro with scaling method of resize so closing the bug.

https://github.com/user-attachments/assets/61c3c17c-decd-4c8b-8550-7335d3076be3
